TimesReactIOS.podspec

```diff
@@ -40,8 +40,8 @@ Pod::Spec.new do |s|
   # React's dependencies
 
   s.dependency 'RNDeviceInfo'
-  s.dependency 'RNSVG'
-  s.dependency 'yoga'
+  s.dependency 'RNSVG', '6.3.1'
```

Comment:
Can we use the values from package json? We already got react-version on this file